git-hooks/commit-msg

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+git secrets --commit_msg_hook -- "$@"
```

Comment:
Suggestion for future PR: add some instructions to the Readme that mentions how to install the git secret hooks locally and use the install hooks script. Then Dockstore engineers could have git secrets running/set up locally, and that step can make its way into an (the?) onboarding checklist.

Reply:
The intention of this PR is to make it so you don't have to install the hooks. If you use the maven machinery, the hooks will be setup for this repo with the settings we advise. `git-secrets` just needs to be on the path somewhere for this to work, which is an improvement over having to install and manage them individually. I added instructions for that to the readme.

Reply:
Okay, that clears things up, I wasn't thinking about it correctly before